### ISSUE TITLE
"Play next" option on context menu of a song in the playing queue view

### ISSUE THREAD

**Issue by Hivenfour** [fast4x/RiMusic#908]
### Description

I want to play a song that is already in the queue to be played, but I want to move it to the next position of the currently playing song

### Suggested Solution

As youtube music, an option to "play next" in context menu of a song, in the currently playing queue view

**Comment by fast4x** [comment]
Is just in context menu Play next, look at image
![Screenshot_20240302_162933](https://github.com/fast4x/RiMusic/assets/1484476/a9a6c630-cbde-4f0a-9d46-c2b913dc5fed)

**Comment by Lolozweipunktnull** [comment]
I think the creator is referring to the context menu of songs in the queue (the songs that will be played next). There is no such option, you have to drag the song to the top, which can sometimes be annoying.
![Screenshot_20240303-202720125 (1).png](https://github.com/fast4x/RiMusic/assets/111334483/9c762c2a-954e-4680-a20d-93876559120f)

**Comment by Hivenfour** [comment]
Exactly! I meant what Lolozweipunktnull said (sorry for the inaccurate description)

**Comment by fast4x** [comment]
Implemented, available from version 0.6.25
